### ISSUE TITLE
Bug Report: Unknown Device Reported

### ISSUE THREAD

**Issue by home-automate** [bloomkd46/homebridge-XfinityHome#30]
### Describe The Bug:

The following was shown at Homebridge Startup:

[28/06/2023, 13:43:46] [Xfinity Home] Network Peripheral: Unknown accessory! 
[28/06/2023, 13:43:46] [Xfinity Home] Network Peripheral: Please open an issue about this. 
[28/06/2023, 13:43:46] [Xfinity Home] Network Peripheral: {
  "hardwareId": "*********.net.00:00:00:00:00:00",
  "manufacturer": "XFINITY",
  "macAddress": "000000000000",
  "status": "online",
  "technology": "wifi",
  "_links": {
    "self": {
      "href": "/client/icontrol/device/*********.net.00:00:00:00:00:00"
    }
  },
  "model": "Gateway",
  "deletable": true,
  "deviceType": "peripheral",
  "icontrolModel": "Gateway",
  "properties": {
    "type": "gateway",
    "label": "Network Peripheral"
  },
  "id": "*********.net.00:00:00:00:00:00.0",
  "deviceSubtype": "default",
  "trouble": [],
  "name": "Network Peripheral"
} 
[28/06/2023, 13:43:47]

### To Reproduce:

Start Homebridge with Xfinity 3.3.0 plug-in installed

### Expected behavior:

No unknown accessory in the log

### Logs:

```shell
[28/06/2023, 13:43:46] [Xfinity Home] Network Peripheral: Unknown accessory! 
[28/06/2023, 13:43:46] [Xfinity Home] Network Peripheral: Please open an issue about this. 
[28/06/2023, 13:43:46] [Xfinity Home] Network Peripheral: {
  "hardwareId": "*********.net.00:00:00:00:00:00",
  "manufacturer": "XFINITY",
  "macAddress": "000000000000",
  "status": "online",
  "technology": "wifi",
  "_links": {
    "self": {
      "href": "/client/icontrol/device/*********.net.00:00:00:00:00:00"
    }
  },
  "model": "Gateway",
  "deletable": true,
  "deviceType": "peripheral",
  "icontrolModel": "Gateway",
  "properties": {
    "type": "gateway",
    "label": "Network Peripheral"
  },
  "id": "*********.net.00:00:00:00:00:00.0",
  "deviceSubtype": "default",
  "trouble": [],
  "name": "Network Peripheral"
} 
[28/06/2023, 13:43:47]
```

### Config:

```shell
.
```

### Operating System:

Raspberian

### Node Version:

0

### NPM Version:

0

### Homebridge Version:

0

### Plugin Version:

3.3.0

**Comment by bloomkd46** [comment]
patched 3.4.1

**Comment by home-automate** [comment]
Fixed

![image](https://github.com/bloomkd46/homebridge-XfinityHome/assets/108997613/8f419fa0-d6c1-43e0-bec5-bf875cffe7d1)
